classes/class-p4-loader.php

```diff
@@ -52,6 +52,89 @@ public static function get_instance( $services = [] ) : P4_Loader {
 	private function __construct( $services ) {
 		$this->load_files();
 		$this->load_services( $services );
+		add_filter( 'wp_kses_allowed_html', [ $this, 'filter_allowed_html' ], 10, 2 );
+	}
+
+	/**
+	 * Get allowed tags and add some more
+	 *
+	 * @param string $context context.
+	 * @return mixed
+	 */
+	public function filter_allowed_html( $context = '' ) {
```

Comment:
There is already a function that adds allowed tags at `\P4_Master_Site::set_custom_allowed_attributes_filter`, probably better to add these extra tags there.

Reply:
It is a lot better in there cause it does not spoil for example twitter & youtube embeds.